src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Random randomBall = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int number = randomBall.nextInt(100);
+        Ball ball = new Ball(colorSupplier.getRandomColor(),number);
```

Comment:
According to the checklist, we should not create temporary variables if we can directly return a value from the method. Here, you can return the `Ball` instance directly without assigning it to the `ball` variable.